src/components/Home/Home.js

```diff
@@ -8,20 +8,22 @@ import StatePicker from '../StatePicker/StatePicker';
 import { Route, Switch } from 'react-router-dom';
 import FutureVisits from '../FutureVisits/FutureVisits';
 
+import '../Error/Error.css'
+
 const Home = () => {
 
   const [parks, setParks] = useState([])
   const [parksByState, setParksByState] = useState([])
   const [futureTrips, setFutureTrips] = useState([])
   const [error, setError] = useState('')
-  const [loading, setLoading] = useState('')
 
   useEffect(() =>{ 
     getParksByActivities()
     .then(data => {
      const hikeParks = filterParksForHiking(data)
       setParks(hikeParks)
     })
+    .catch(err => setError('Something went wrong. Please try again later'))
```

Comment:
In getParksByActivities, do you pass your data through an error checking fn? Your catch won't necessarily catch all errors.

Reply:
Added a check for error in my apiCalls file!